plugins/modules/activation_key.py

```diff
@@ -45,6 +45,12 @@
     description:
       - Name of the content view
     type: str
+  content_view_environments:
+    description:
+      - List of content view environments to add to activation key
+      - Ignored if content_view_id and lifecycle_environment_id are specified
+    type: list
+    elements: str
```

Comment:
```suggestion
    elements: str
    version_added: 5.8.0
```

That way docs render this as a "new" parameter and people don't try to use it on older versions